data/projects.json

```diff
@@ -148,6 +148,16 @@
         "level": "basic",
         "site": "",
         "tags": "python-project,personal-assistant,python-assistant"
+    },
+    {
+        "name": "Gorcery-FinderApp",
+        "display": "Gorcery helper using React Native",
+        "description": "A mobile app that assists customers in finding what isle their items are on",
+        "author": "Dustin1864",
```

Comment:
The user name is not exists

Reply:
If you are the owner of the repository then please change the name of the author from @Dustin1864 to @Dustin1386 , in the case of another Author, Please provide an existing username